cognite/client/_api/files.py

```diff
@@ -600,17 +600,19 @@ def download(
         directory: str | Path,
         id: int | Sequence[int] | None = None,
         external_id: str | Sequence[str] | None = None,
+        keep_folder_structure: bool = False,
     ) -> None:
         """`Download files by id or external id. <https://developer.cognite.com/api#tag/Files/operation/downloadLinks>`_
 
         This method will stream all files to disk, never keeping more than 2MB in memory per worker.
-        The files will be stored in the provided directory using the name retrieved from the file metadata in CDF.
-
+        The files will be stored in the provided directory using the file name retrieved from the file metadata in CDF.
+        You can also choose to keep the folder structure from CDF, where the files will be stored in subfolders under directory named after the directory field in CDF.
```

Comment:
Since the original text (and the field name used by CDF) uses _directory_ (for folder), I suggest we stick to this name (also in the argument name, i.e. `keep_directory_structure` or `keep_dir_structure`